continuous_integration/environment-latest.yaml

```diff
@@ -1 +0,0 @@
-environment-3.8.yaml
```

Comment:
Let's keep this file around as we use it in our user documentation (see the discussion in https://github.com/dask/dask/pull/6399)

Reply:
reverted